scripts/pylib/twister/twisterlib.py

```diff
@@ -3061,6 +3062,8 @@ def __init__(self, board_root_list=[], testsuite_roots=[], outdir=None):
 
         self.modules = []
 
+        self.timestamp = datetime.now().isoformat()
```

Comment:
Hi, not sure if this is a problem. This timestamp seems like in testplan level, so we will see all testsuites have the same timestamps.

Reply:
I do not think the idea behind the timestamp is to get time accurate data for when exactly something was run, it is about the overall testrun. Anyways, this can always be changed if needed.

Reply:
ok, got it, thanks.